public_dropin_gpu_environments/nemollm_inference_ms/start_server.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/sh
+# Copyright 2024 DataRobot, Inc. and its affiliates.
+#
+# All rights reserved.
+# This is proprietary source code of DataRobot, Inc. and its affiliates.
+#
+# Released under the terms of DataRobot Tool and Utility Agreement.
+echo "Starting Custom Model environment with NIM"
+set -e
+
+if [ -z "${$MODEL_NAME}" ]; then
+    echo "The MODEL_NAME must be set in runtime parameters"
+fi
```

Comment:
Runtime params don't work like this (it's JSON encoded and has the prefix `MLOPS_RUNTIME_PARAM_...` or something like that (we shouldn't rely on these details though) -- the only official way to support this is using the DRUM code though.

Reply:
Why not just hard code the model name? From what I can tell, the only thing it is used for is to simulate the OpenAI API (which requires a model name because the same library can talk to a multitude of different models OpenAI currently has but with DRUM we will only be serving one model). As long as this startup code, and the _drum proxy code_ agree on the model name, I think it can be hard coded right?

Reply:
> Why not just hard code the model name? 

Ah, cool. I didn't knew that you can set anything as a model name. I'll hardcode model name then for NIM.

For Triton, you have to specify the correct model name for API requests, which I'm reading from the `config.pbtxt`.